source/deployment/deployContracts.ts

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
```

Comment:
TS files can't be directly executed by `node` so this line isn't correct and should be left out.

Reply:
Ok, I wasn't sure if we wanted the JS file generated from the TS file to have the shebang line at the top.

Reply:
Oh, will this land in the generated JS files?  If so I guess this is fine.  I tend to not execute scripts directly (always `node ...`) but I know a lot of other people do and if this makes it to the JS file then that seems valuable to those people.

Reply:
Yeah, I checked the generated JS file, and it does show up there.  I'll go ahead and leave it in.